quickwit/quickwit-actors/src/lib.rs

```diff
@@ -91,21 +91,25 @@ fn heartbeat_from_env_or_default() -> Duration {
         // We use a shorter heartbeat to reduce the time running unit tests.
         return Duration::from_millis(500);
     }
-    if let Ok(actor_hearbeat_secs_str) = std::env::var("QW_ACTOR_HEARTBEAT_SECS") {
-        if let Ok(actor_hearbeat_secs) = actor_hearbeat_secs_str.parse::<NonZeroU64>() {
-            info!("Set the actor heartbeat to {actor_hearbeat_secs} seconds.");
-            return Duration::from_secs(actor_hearbeat_secs.get());
-        } else {
+    match std::env::var("QW_ACTOR_HEARTBEAT_SECS") {
```

Comment:
Well-intentioned, but at the end of the day, it will only be used by advanced users, most likely us exclusively. So I'd panic and call it a day.

Reply:
Agreed but... the program will not exit with a panic in the initialization. We could parse it at the beginning as we do for the config.